site/content/en/docs/manual/advanced/xml_format.md

```diff
@@ -229,12 +229,12 @@ Example:
     </points>
     <tag label="good_frame" source="manual">
     </tag>
-    <skeleton label="s1" occluded="0" source="manual" points="" rotation="0.00" z_order="0">
-      <points label="1" occluded="0" source="manual" outside="0" points="54.47,94.81" z_order="0">
+    <skeleton label="s1" occluded="0" source="manual" z_order="0">
```

Comment:
I've set occluded for track:
![image](https://user-images.githubusercontent.com/41117609/217481656-8c7aa2f4-530e-4d56-8841-ddff610bd1f2.png)
but
![image](https://user-images.githubusercontent.com/41117609/217481913-87f23ba1-6590-494e-9da3-053b34a02707.png)
Is it expected behaviour?

Reply:
This field should be only in parts of the skeleton. Fixed it, thanks.